### PR TITLE
chore: add Make commands for easier development

### PR DESCRIPTION
**Pull request by trowik**
Added some commands to make the development of the app a little bit easier.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,26 +1,57 @@
...
+	@docker-compose exec backend sh -c "poetry run black --check . && poetry run flake8"
+
+.PHONY: format-code
+format-code: ## Format the backend code
+	@docker-compose exec backend sh -c "poetry run black . && poetry run isort ."
+
+.PHONY: test
 test: ## Test the project
-	@docker-compose exec backend sh -c "poetry run black --check . && poetry run flake8 && poetry run pytest --no-cov-on-fail --cov"
+	@docker-compose exec backend poetry run pytest --no-cov-on-fail --cov
```
Comment (open-dynaMIX) [inline Makefile:25]: YMMV, but I like it if `make test` also lints the code.

File: timed/settings.py [hunk 2]
```diff
@@ -61,6 +61,7 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
     "django.contrib.staticfiles",
     "rest_framework",
     "django_filters",
+    "django_extensions",
```
Comment (open-dynaMIX) [inline timed/settings.py:64]: `django-extensions` is a dev dependency (and should be), so this is not gonna work in prod.

You should add a block like this below `INSTALLED_APPS`:

```python
if ENV == "dev":
    INSTALLED_APPS.append("django_extensions")
```

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (trowik) [reply Makefile]: Ok, changed it back

File: timed/settings.py (hunk 2 above)
Comment (trowik) [reply timed/settings.py]: Done. Thanks a lot.

File: Makefile [hunk 3]
```diff
@@ -1,26 +1,57 @@
...
+	@docker-compose exec backend sh -c "poetry run black --check . && poetry run flake8"
+
+.PHONY: format-code
+format-code: ## Format the backend code
+	@docker-compose exec backend sh -c "poetry run black . && poetry run isort ."
+
+.PHONY: test
 test: ## Test the project
-	@docker-compose exec backend sh -c "poetry run black --check . && poetry run flake8 && poetry run pytest --no-cov-on-fail --cov"
+	@docker-compose exec backend sh -c "poetry run black . && poetry run isort . && poetry run pytest --no-cov-on-fail --cov
```
Comment (open-dynaMIX) [inline Makefile:25]:
```suggestion
	@docker-compose exec backend sh -c "poetry run black . && poetry run flake8 . && poetry run pytest --no-cov-on-fail --cov
```

File: timed/settings.py [hunk 4]
```diff
@@ -74,6 +74,9 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
     "timed.subscription",
 ]
 
+if ENV == "dev":
```
Comment (open-dynaMIX) [inline timed/settings.py:77]: This is really set to `dev` in dev?

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Some more nitpicks / questions ;)

File: timed/settings.py (hunk 4 above)
Comment (trowik) [reply timed/settings.py:77]: Yes, it's written in `README.md`

>## Development

To get the application working locally for development, make sure to create a file `.env` with the following content:

```
ENV=dev
DJANGO_OIDC_CREATE_USER=True
````

Comment (open-dynaMIX) [reply timed/settings.py:77]: :+1:

**Review by open-dynaMIX — APPROVED** [review]

**Pull request merged**